### ISSUE TITLE
Action to PKC nodes not working

### ISSUE THREAD

**Issue by scb147** [croneter/PlexKodiConnect#835]
I am using LibreElec 9.0.1 Generic.x86_64 that has Kodi Leia 18.1.  PKC version is 2.7.10.  I am trying to use the Pellucid skin, 2.1.0.

I have PKC setup and syncing to my PMS. I am trying to setup my action for the Movies menu item. From the skin setings, I navigate to Menus-->Edit home menu. I select Movies and then the Change action option. In the "Choose Item For Menu" dialog, I select Add-on-->Video add-on-->PlexKodiConnect-->Movies-->Movies option. I exit the skin settings.

From the main menu, I select Movies. I then see a screen that says:
0%
Scanning Media Information
Loading media information from files...
videodb://movies/titles/?xsp%7b%22or

This menu goes away, and then in the list of movies, I have a number instead of the movie title.

If I navigate to Addons-->PlexKodiConnect-->Movies-->Movies, I get the listings of movies that I would expect. 

I have also tried this with the Titan-MOD and Embuary skins, both with the same issues.

**Comment by dazedcrazy** [comment]
```
Add-on-->Video add-on-->PlexKodiConnect-->Movies-->Movies (More...)-->All Movies-->Create menu item here
```
That will give you a proper list of your movies

**Comment by scb147** [comment]
Yeah, I get that, but croneter posted this over on the plex.tv forum:

> The More... video node for libraries basically mirrors the different Plex views of a library. These nodes are a bit slower to load since the PMS needs to be contacted (most other nodes are directly “assembled” by Kodi and thus a bit faster).

So this node not working seems like a reasonable issue.

**Comment by dazedcrazy** [comment]
I figured it out. For some reason PKC or skinshortcuts is adding node as a music node, it looks like this
![image](https://user-images.githubusercontent.com/9027988/56456282-01d21200-6362-11e9-9aca-2f6934786e7c.png)
When it should look like this
![image](https://user-images.githubusercontent.com/9027988/56456287-10b8c480-6362-11e9-9b8e-51524fa233f1.png)
2 ways you can fix this. First way is the easiest, when adding the node scroll to the bottom and you'll see `Custom item > ` when you hit that you'll get a window like this
![image](https://user-images.githubusercontent.com/9027988/56456316-5ffef500-6362-11e9-8df0-9705b76082b8.png)
Just change the Music to Videos
Second way is to go into `Kodi\userdata\addon_data\script.skinshortcuts` and find 
`skin.YOUR-SKIN-mainmenu.DATA.xml` and edit it there

**Comment by croneter** [comment]
# Quick Fix
This is actually a Skinshortcuts bug 😞. Solution: When choosing the widget, be sure to navigate as follows:
`Video Library` -> `<name of your Plex section>` -> `Node that you'd like`
